hilti/visualize_trajectories.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import open3d as o3d
+from colour import Color
+
+import argparse
+import os
+from typing import List
+
+from hilti.utils.utils import (
+    get_camera_to_lidar,
+    get_lidar_to_imu,
+    get_sorted,
+    read_pose,
+)
+
+
+def get_trajectory(path: str, color: List) -> List:
```

Comment:
I would pass here parameter, called "normalize"

Then, you aggregate poses as they are to list `poses`, and
```
if normalize:
   poses = [np.linalg.inv(poses[0]) @ pose for pose in poses]
```

Then, no need to check path

Reply:
Then you can easily generate `meshes = [o3d.geometry.TriangleMesh.create_coordinate_frame().transform(pose) for pose in poses]`

Reply:
Fixed in 2d02b62475a1320fa749ba3f40cb33569209eb26